src/webview.rs

```diff
@@ -82,6 +83,11 @@ impl Window {
                     w
                 );
             }
+            EmbedderMsg::LoadStart => {
+                if let Some(init_script) = &self.init_script {
+                    execute_script_async(&sender, &webview_id, init_script);
```

Comment:
Why doesn't it just use usercript?

Reply:
I want to use it later on for the webview controller's APIs, for the CLI usage, we can also add in userscript along side this, since that one needs to take a file instead of a string